docs/advanced/naming.md

```diff
@@ -1,190 +1,245 @@
+Do not include the [type of release](/guides/quality/#types-of-releases) and [where it is sourced from](/guides/quality/#blu-ray-vs-web)
+:   - It can be harder to determine which release is the better one
+    - File 2 does not list the video codec/resolution, audio codec, "Blu-ray/WEB" source, dual audio, etc.
 
-```
-Anime Name - 18.5 - S00E05 - Title of the Episode
-```
+May be too long in length
+:   - Many programs do not support file paths longer than 256 characters
+    - File 2 includes episode chapters in the file name, totaling 159 characters alone
 
-- Another good example, this tells us the name of the anime, where it lands chronologically, what episode it is with reference to databases and the name of the episode to aid with identifying the episode without much effort.
+Use an inefficient/invalid naming scheme
+:   - File 1 uses absolute numbering for episodes. *We recommend using the `SXXEXX` format instead*
+    - File 3 uses brackets to hold the season and episode number (`[S02,E00]`). *As a result, some programs may incorrectly parse this as a release group*
 
-#### More Examples
+Alter or uses an incorrect title
+:   - Some programs may fetch incorrect title metadata
+    - File 3 uses the incorrect order for the show title (`From Me to You Kimi Ni Todoke` instead of `Kimi ni Todoke - From Me to You`)
+    - File 4 does not include the show year. *As a result, media servers may fetch incorrect metadata ([JoJo's Bizarre Adventure](https://thetvdb.com/series/jojos-bizarre-adventure) is detected instead of [JoJo's Bizarre Adventure (2012)](https://thetvdb.com/series/jojos-bizarre-adventure-2012))*
 
-Bad, do not use:
+Using good naming schemes, it could look like the following:
 
-```
-[Group] Anime Name - S02OVA02.mkv
-[Group] Anime Name - 18.5 [BD 1080p FLAC] [CRC].mkv
-[Group] Anime Name - 06.5 (SP) [BD 1080p FLAC].mkv
-[Group] Anime Name - Title of the Episode [CRC].mkv
-[Group] Anime Name - 17.5 (OVA) [CRC].mkv
-```
+![Releases using good naming schemes](/static/advanced/naming/releases-good-naming.png)
 
-Good names:
+### Recommended Schemes
 
-```
-Anime Name - 06.5 - S00E04 - Title of the Episode (BD 1080p HEVC FLAC) [Dual Audio] [CRC32] [Group].mkv
-Anime Name - S00E01 - Title of the Episode (AMZN WEB-DL 1080p H.264 EAC3) [Dual Audio] [Group].mkv
-Anime Name - 18.5 - S00E05 - Title of the Episode (BD 1080p HEVC FLAC) [Dual Audio]-Group.mkv
-Anime Name (2022) - S00E01 - Title of the Episode (BD Remux 1080p HEVC FLAC) [Dual Audio]-Group.mkv
-Anime.Name.S00E09.Title.of.the.Episode.1080p.BluRay.Opus2.0.x264-Hi10P-Group.mkv
-```
+A good file name allows a viewer to easily understand what the file is and what it contains. See the recommended variants below:
+
++++ :icon-file-code: Variant 1
+
+![Example file name using variant 1](/static/advanced/naming/variant-1.png)
+
+!!!warning
+Spaces and special characters (i.e. apostrophes `'`, commas `,`) should be replaced with periods `.` instead.
```

Comment:
spaces and commas should be replaced with periods, apostrophes should simply be stripped